ecosystem/sep-0031.md

```diff
@@ -194,6 +194,8 @@ Content-Type: application/json
 
 {
   "amount": 100,
+  "asset_code": "USD",
+  "asset_issuer": "GDRHDSTZ4PK6VI3WL224XBJFEB6CUXQESTQPXYIB3KGITRLL7XVE4NWV",
```

Comment:
Did you folks also consider using the single field string representation that's defined in SEP-11? We've started using that in some places instead of separate fields.

Reply:
We didn't, but I don't I see the value in using that notation. The same info is communicated without requiring the receiver to parse the string. And the SEP-11 notation requires the client specify an issuer, where in this spec the issuer is optional.

Reply:
@leighmcculloch @jacekn was there a reason for not using `asset: assetcode:assetissuer` which is the canonical representation for assets? we are moving towards this representation in Horizon

Reply:
btw, we are trying to push for SEP-11 notation on Horizon and it would be great if we start doing the same in the SEPs

Reply:
The main value is consistency so that assets are represented the same everywhere. It means that when integrating with different SEPs developers can pass the same strings without needing to format differently. But the ability to specify only the code makes it interesting and complicated, SEP-11 doesn't support that well because you wouldn't be able to distinguish between native the asset code or native the asset type.

Reply:
@leighmcculloch 
> SEP-11 doesn't support that well because you wouldn't be able to distinguish between native the asset code or native the asset type.

Why not?

Reply:
@abuiles If @JakeUrban's user wishes to specify exactly the native asset they would specify it as `"asset": "native"` according to SEP-11. If the user wishes to specify any issued asset that has the code `native` and any issuer and omitted the `:<issuer>` they would also specify it as `"asset": "native"`. There'd be no way for the server to distinguish if the request was specifying the native asset or an issued asset with code `native`.

Reply:
@leighmcculloch ok, makes sense! I missed the last bit on Jake's comment. :)